File: openex-platform/openex-api/src/Entity/DryinjectStatus.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;


/**
 * @ORM\Entity()
 * @ORM\Table(name="dryinjects_statuses")
 */
class DryinjectStatus
{
    /**
     * @ORM\Id
     * @ORM\Column(type="string")
     * @ORM\GeneratedValue(strategy="CUSTOM")
     * @ORM\CustomIdGenerator("doctrine.uuid_generator")
     */
    protected $status_id;

    /**
     * @ORM\OneToOne(targetEntity="Dryinject", inversedBy="dryinject_status")
     * @ORM\JoinColumn(name="status_dryinject", referencedColumnName="dryinject_id", onDelete="CASCADE")
     */
    protected $status_dryinject;

    /**
     * @ORM\Column(type="string", nullable=true)
     */
    protected $status_name;

    /**
     * @ORM\Column(type="text", nullable=true)
     */
    protected $status_message;

    /**
     * @ORM\Column(type="datetimetz", nullable=true)
     */
    protected $status_date;

    /**
     * @ORM\Column(type="integer", nullable=true)
     */
    protected $status_execution;

    public function getStatusId()
    {
        return $this->status_id;
    }

    public function setStatusId($id)
    {
        $this->status_id = $id;
        return $this;
    }

    public function getStatusDryinject()
    {
        return $this->status_dryinject;
    }

    public function setStatusDryinject($dryinject)
    {
        $this->status_dryinject = $dryinject;
        return $this;
    }

    public function getStatusName()
    {
        return $this->status_name;
    }

    public function setStatusName($name)
    {
        $this->status_name = $name;
        return $this;
    }

    public function getStatusMessage()
    {
        return $this->status_message;
    }

    public function setStatusMessage($message)
    {
        $this->status_message = $message;
        return $this;
    }

    public function getStatusDate()
    {
        return $this->status_date;
    }

    public function setStatusDate($date)
    {
        $this->status_date = $date;
        return $this;
    }

    public function getStatusExecution()
    {
        return $this->status_execution;
    }

    public function setStatusExecution($status_execution)
    {
        $this->status_execution = $status_execution;
        return $this;
    }
}

File: openex-player/src/main/java/io/openex/database/model/DryInjectStatus.java
package io.openex.database.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "dryinjects_statuses")
public class DryInjectStatus implements Base {
    @Id
    @Column(name = "status_id")
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
    @JsonProperty("status_id")
    private String id;

    @Column(name = "status_name")
    @JsonProperty("status_name")
    private String name;

    @Column(name = "status_message")
    @JsonProperty("status_message")
    private String message;

    @Column(name = "status_date")
    @JsonProperty("status_date")
    private Date date;

    @Column(name = "status_execution")
    @JsonProperty("status_execution")
    private Integer executionTime;

    @OneToOne
    @JoinColumn(name = "status_dryinject")
    @JsonIgnore
    private DryInject<?> dryInject;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Integer getExecutionTime() {
        return executionTime;
    }

    public void setExecutionTime(Integer executionTime) {
        this.executionTime = executionTime;
    }

    public DryInject<?> getDryInject() {
        return dryInject;
    }

    public void setDryInject(DryInject<?> dryInject) {
        this.dryInject = dryInject;
    }
}
